gem/styles/gem-chhaajaa/modules/reactions.scss

```diff
@@ -1,16 +1,20 @@
 .reaction-questions {
+  margin: 0 5% 0 0;
   @media (min-width: 1200px) {
-    position: absolute;
-    right: 8%;
-    top: 10px;
+    float: right;
+    max-height: 85px;
+  }
+  &__form {
```

Comment:
what does this do?

Reply:
@codiebeulaine This hide the likes counter.

Reply:
@Mitso what likes counter? Can you keep it in there? for now, because I'm making changes on my side